docs/articles/azure/azs-how-create-key-vault-ps.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to create a Key Vault - PowerShell | UKCloud Ltd
+description: Provides help for creating a Key Vault on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a Key Vault - PowerShell
+toc_fullpath: Users/How To/azs-how-create-key-vault-ps.md
+toc_mdlink: azs-how-create-key-vault-ps.md
+---
+
+# How to create a Key Vault in Azure Stack with PowerShell
+
+## Overview
+
+Key Vault in Azure Stack helps safeguard cryptographic keys and secrets that cloud applications and services use. By using Key Vault, you can encrypt keys and secrets.
+
+The following process shows you how to setup a vault within Azure Stacks Key Vault, store a secret in the vault and how to extract the secret.
+
+## Prerequisites
+
+Ensure your Powershell environment is setup as detailed in [Configure PowerShell Environment and Azure Stack Modules](https://docs.ukcloud.com/articles/azure/azs-how-configure-powershell-users.html).

```

Comment:
Make sure you use the correct title of the article so that users know they've ended up in the right place: Configure the Azure Stack user's PowerShell environment

Also, use a relative value for the link (and point to the markdown file). In this case, it just needs to be: azs-how-configure-powershell-users.md

Reply:
Fixed